recipes/ribasim/meta.yaml

```diff
@@ -0,0 +1,79 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Ping.

Reply:
Thanks for the nudge.

Removed the comments and followed closer what grayskull produces